docs/data/date-pickers/date-calendar/DateCalendarServerRequest.js

```diff
@@ -38,7 +38,7 @@ function ServerDay(props) {
   const { highlightedDays = [], day, outsideCurrentMonth, ...other } = props;
 
   const isSelected =
-    !props.outsideCurrentMonth && highlightedDays.indexOf(props.day.date()) > 0;
+    !props.outsideCurrentMonth && highlightedDays.indexOf(props.day.date()) >= 0;
```

Comment:
You could also consider using `some` to avoid these silly one-off mistakes. 🙈 🤔 
```suggestion
    !props.outsideCurrentMonth &&
    highlightedDays.some((highlightedDay) => highlightedDay === props.day.date());
```